_includes/sections/password-managers.html

```diff
@@ -67,6 +67,23 @@ <h1 id="pw" class="anchor"><a href="#pw"><i class="fas fa-link anchor-icon"></i>
   googleplay="https://play.google.com/store/apps/details?id=com.lesspass.android"
 %}
 
+{%
+  include cardv2.html
+  title="Pass - Self-Host"
+  image="/assets/img/svg/3rd-party/pass.svg"
+  description="<strong>Pass</strong> is a very simple password store that keeps passwords using gpg2 encrypted files inside a simple directory tree residing at ~/.password-store. It has a simple terminal interface where the user can perform the usual actions, and some plugins can be installed for extra functionalities."
```

Comment:
```suggestion
  description="<strong>Pass</strong> is a very simple password store that keeps passwords using gpg2 encrypted files inside a simple directory tree residing at <code>~/.password-store</code>. It has a simple terminal interface where the user can perform the usual actions, and some plugins can be installed for extra functionalities."
```

I am not entirely sure on this description though, are Windows users likely to be familiar with `~` ?